main.go

```diff
@@ -60,25 +60,31 @@ func main() {
 
 	app.Action = func() {
 		initLogs(os.Stdout, os.Stdout, os.Stderr)
-		transport := &http.Transport{Dial: proxy.Direct.Dial}
+		transport := &http.Transport {
+			Dial: proxy.Direct.Dial,
+			ResponseHeaderTimeout: 10 * time.Second,
+			MaxIdleConnsPerHost: 100,
```

Comment:
How does this number come up?

Is the idea that we have about 150-200 services then on a single host probably there isn't more then 100 running (probably 30-40)?